hstaxe/axesrc/axetasks.py

```diff
@@ -681,15 +681,15 @@ def gol2af(grism='',
 
     # run GOL2AF
     gol2af = axelowlev.aXe_GOL2AF(grism, config,
-                                  back=back,
                                   mfwhm=mfwhm,
                                   orient=orient,
                                   slitless_geom=slitless_geom,
                                   exclude=exclude,
                                   lambda_mark=lambda_mark,
                                   dmag=dmag,
                                   in_gol=in_gol,
-                                  out_af=out_af)
+                                  out_af=out_af,
+                                  back=back)
```

Comment:
Was there any reason this needed to go last? Order of keyword arguments shouldn't matter, right?

Reply:
That's the weirdest thing. I agree, the order of the keyword arguments **shouldn't** matter, but when I tested it, `back` only got passed through when I moved it out from the first args! Isn't that weird? I thought maybe there was some python behavior/rule that I wasn't aware of 🤷

Reply:
Very strange 🤔

Reply:
Keyword arguments can also be passed by position.  Hopefully this function isn't called by positional arguments, but you could add a "*" in the function signature between the positional and keyword arguments to force anyone using the function to use only keyword inputs.